otel-agent/ecs-ec2/logging.yaml

```diff
@@ -48,5 +75,13 @@ service:
       processors:
         - resourcedetection
         - ecsattributes
+        - filter/ecslogs
+        - transform/logs
+      exporters:
+        - coralogix
+
+    traces:
+      receivers:
+        - otlp
```

Comment:
Is it intended to add `traces` pipeline to this logging config?

Reply:
removed